hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/execution/bulkinsert/SpatialCurveSortPartitionerBase.java

```diff
@@ -78,6 +78,8 @@ protected Dataset<Row> reorder(Dataset<Row> dataset, int numOutputGroups) {
 
   @Override
   public boolean arePartitionRecordsSorted() {
-    return true;
+    //The data is sorted using a function that maps multiple columns into a single dimension.
```

Comment:
nit: Please make sure comments are formatted in-line with the rest of the code-base